README.md

```diff
@@ -25,6 +27,10 @@ config :pool_max, :validate => :number, :default => 50
 # Max number of concurrent connections to a single host. Defaults to 25
 config :pool_max_per_route, :validate => :number, :default => 25
 
+# Specify this if you want to keep the TCP connection open between requests
+# using HTTP Keep-Alive. Defaults to 0 (off)
+config :keepalive, :validate => :number, :default => 0
```

Comment:
>  keepalive (Boolean/Fixnum) — default: true —
> Whether to allow connections to be reused. Defaults to true. If an integer, then connections will be kept alive for this long when Connection: keep-alive is sent, but no Keep-Alive header is sent.

This is confusing. "kept alive for this long" - how long? This API feels weird, I wonder if we shouldn't expose it? Alternately, change the setting to be described as "If 0, no keepalive is used. If greater than 0, means the number of [mystery time unit] a connection will be kept alive for reuse."